File: serve.rb
require 'webrick'
server = WEBrick::HTTPServer.new(
  Port: 8080,
  DocumentRoot: '/Users/arturkrivichev/Downloads/DM/Code',
  Logger: WEBrick::Log.new('/dev/null'),
  AccessLog: []
)
trap('INT') { server.shutdown }
server.start

File: serve.py
import http.server
import os

os.chdir("/Users/arturkrivichev/Downloads/DM/Code")

class Handler(http.server.SimpleHTTPRequestHandler):
    def log_message(self, format, *args):
        pass  # suppress logs

http.server.HTTPServer(("", 3000), Handler).serve_forever()
